packages/cli/test/mocks/team.ts

```diff
@@ -1,6 +1,17 @@
 import chance from 'chance';
 import { client } from './client';
 
+type Team = {
+  id: string;
+  slug: string;
+  name: string;
+  creatorId: string;
+  created: string;
+  avatar: null;
+};
+
+let teams: Team[] = [];
```

Comment:
The mock here wasn't set up to allow for multiple teams. I adjusted it for the use of the current tests, but we should have a better mocking strategy in general for these kinds resources.